jax/_src/scipy/stats/multinomial.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import scipy.stats as osp_stats
+from jax import lax
+from jax._src.numpy import lax_numpy as jnp
+from jax._src.numpy.util import _wraps, _promote_args_inexact, _promote_args_numeric
+from jax._src.scipy.special import gammaln, xlogy
+
+@_wraps(osp_stats.multinomial.logpmf, update_doc=False)
+def logpmf(x, n, p):
+  """JAX implementation of scipy.stats.multinomial.logpmf."""
+  p, = _promote_args_inexact("multinomial.logpmf", p)
+  x, n = _promote_args_numeric("multinomial.logpmf", x, n)
+  if not (jnp.issubdtype(x.dtype, jnp.integer) and jnp.issubdtype(n.dtype, jnp.integer)):
```

Comment:
nitpick: `x` and `n` have the same dtype here (this is always the case after `_promote_args_*`, so no need to check both.